scripts/deploy-kubefed.sh

```diff
@@ -45,10 +45,10 @@ function deploy-with-helm() {
 
   local cmd
   if [[ "${NAMESPACED}" ]]; then
-    cmd="$(helm-deploy-cmd kubefed-${NS} ${NS} ${repository} ${image} ${tag}) --create-namespace"
+    cmd="$(helm-deploy-cmd kubefed-${NS} ${NS} ${repository} ${image} ${tag})"
     cmd="${cmd} --set global.scope=Namespaced"
   else
-    cmd="$(helm-deploy-cmd kubefed ${NS} ${repository} ${image} ${tag}) --create-namespace"
+    cmd="$(helm-deploy-cmd kubefed ${NS} ${repository} ${image} ${tag})"
```

Comment:
Doe helm automatically create the namespace, if one does not exist already for us now?